runtime/lua/provider/health.lua

```diff
@@ -657,10 +657,76 @@ local function virtualenv()
   end
 end
 
+local function ruby()
+  start('Ruby provider (optional)')
+
+  if disabled_via_loaded_var('ruby') then
+    return
+  end
+
+  if not executable('ruby') or not executable('gem') then
+    warn(
+      '`ruby` and `gem` must be in $PATH.',
+      'Install Ruby and verify that `ruby` and `gem` commands work.'
+    )
+    return
+  end
+  info('Ruby: ' .. system({ 'ruby', '-v' }))
+
+  local ruby_detect_table = vim.fn['provider#ruby#Detect']()
+  local host = ruby_detect_table[1]
+  if is_blank(host) then
+    warn('`neovim-ruby-host` not found.', {
+      'Run `gem install neovim` to ensure the neovim RubyGem is installed.',
+      'Run `gem environment` to ensure the gem bin directory is in $PATH.',
+      'If you are using rvm/rbenv/chruby, try "rehashing".',
+      'See :help g:ruby_host_prog for non-standard gem installations.',
+      'You may disable this provider (and warning) by adding `let g:loaded_ruby_provider = 0` to your init.vim',
+    })
+    return
+  end
+  info('Host: ' .. host)
+
+  local latest_gem_cmd = (iswin and 'cmd /c gem list -ra "^^neovim$"' or 'gem list -ra ^neovim$')
+  local latest_gem = system(vim.fn.split(latest_gem_cmd))
+  if shell_error() or is_blank(latest_gem) then
+    error(
+      'Failed to run: ' .. latest_gem_cmd,
+      { "Make sure you're connected to the internet.", 'Are you behind a firewall or proxy?' }
+    )
+    return
+  end
+  local gem_split = vim.fn.split(latest_gem, [[neovim (\|, \|)$]])
+  if vim.tbl_contains(gem_split, 0) then
+    latest_gem = 'not found'
+  end
```

Comment:
`vim.tbl_contains` checks table values, not keys. Judging by corresponding Vimscript, it should be something like `latest_gem = gem_split[0] or 'not found'`.

Reply:
Oh wow, the second argument for `get()` is the index/key. OK thanks!